frontend/src/pages/EditUserPage.tsx

```diff
@@ -0,0 +1,273 @@
+import React, { useEffect } from 'react';
+import {
+  Container,
+  Row,
+  Col,
+  Spinner,
+  Button,
+  Navbar,
+  FormControl,
+  FormGroup,
+  FormLabel,
+  InputGroup,
+  ButtonGroup,
+  ButtonToolbar,
+} from 'react-bootstrap';
+import { fetchUserDetail, getUpdateUserAction } from 'actionCreators/usersActionCreator';
+import { useDispatch, useSelector } from 'react-redux';
+import { RootState } from 'reducers/rootReducer';
+import { IUserDetails } from 'actions/adminActions';
+import _ from 'lodash';
+import { Formik, ErrorMessage, Form } from 'formik';
+import { UserUpdateSchema } from 'utils/YupSchema';
+import { IUserDetailParams } from 'constants/API';
+import { ILookupCodeState } from 'reducers/lookupCodeReducer';
+import { ILookupCode } from 'actions/lookupActions';
+import * as API from 'constants/API';
+import './EditUserPage.scss';
+
+const EditUserPage = (props: IUserDetailParams) => {
+  useEffect(() => {
+    dispatch(fetchUserDetail({ id: '41a46ea3-e815-4272-9a17-693ba2409841' }));
+  }, []);
+
+  const dispatch = useDispatch();
+
+  const lookupCodes = useSelector<RootState, ILookupCode[]>(
+    state => (state.lookupCode as ILookupCodeState).lookupCodes,
+  );
+
+  const agencies = _.filter(lookupCodes, (lookupCode: ILookupCode) => {
+    return lookupCode.type === API.AGENCY_CODE_SET_NAME;
+  });
+  const roles = _.filter(lookupCodes, (lookupCode: ILookupCode) => {
+    return lookupCode.type === API.ROLE_CODE_SET_NAME;
+  });
+
+  const user = useSelector<RootState, IUserDetails>(state => state.GET_USER_DETAIL as IUserDetails);
+
+  const getAgencyIdToPut = (name: any) => {
+    let idToSend = '';
+    agencies.map(val => {
+      if (name === val.name) {
+        idToSend = val.id;
+      }
+    });
+    return idToSend;
+  };
+
+  const getRoleIdToPut = (name: any) => {
+    let idToSend = '';
+    roles.map(val => {
+      if (name === val.name) {
+        idToSend = val.id;
+      }
+    });
+    return idToSend;
+  };
+
+  // Arrays below are used to add the role/agency from the dropdown later in code
+  let agenciesToUpdate: any[] = user.agencies;
+  let rolesToUpdate: any[] = user.roles;
+
+  const initialValues = {
+    username: user.username,
+    firstName: user.firstName,
+    lastName: user.lastName,
+    email: user.email,
+    displayName: user.displayName,
+    isDisabled: false,
+    rowVersion: user.rowVersion,
+    emailVerified: false,
```

Comment:
I don't think `emailVerified` was in other userDetail object declarations above.